content/en/modules/nibabel/index.md

```diff
@@ -0,0 +1,136 @@
+<!-- This is an html comment and this won't appear in the rendered page. You are now editing the "content" area, the core of your description. Everything that you can do in markdown is allowed below. We added a couple of comments to guide your through documenting your progress. -->
+
+## Information
+
+The estimated time to complete this training module is 1h30m for the basic content and 1h for the optional advanced content.
+
+The prerequisites to take this module are:
+ * the [introduction to the terminal](/modules/introduction_to_terminal) module.
+ * the [DataLad](/modules/datalad) module.
+
+If you have any questions regarding the module content please ask them in the relevant module channel on the school Discord server.
+If you do not have access to the server and would like to join, please send us an email at school [dot] brainhack [at] gmail [dot] com.
+
+## Resources
+
+We will watch the presentation **NiBabel: Neuroimaging data and file structures in Python** from Neurohackademy 2020 by Chris Markiewicz.
+
+The original presentation and the practical notebook can be found [here](https://github.com/effigies/nibabel-presentations).
+In this exercise, we will use the notebook in the brainhack school repository. Follow the instructions below to run both. 
+
+### Downlad data and jupyter notebook
+
+You will need to install the example dataset with [DataLad](/modules/datalad).
+
+```bash
+# download open neuro dataset
+mkdir ~/brainhackschool_nibabel
+cd ~/brainhackschool_nibabel
+mkdir data
+cd data
+datalad install ///openneuro
+cd openneuro
+datalad get -r ds000114/sub-01
+
+cd ~/brainhackschool_nibabel/data
+datalad install git@github.com:OpenNeuroDerivatives/ds000228-fmriprep.git
+cd ds000228-fmriprep
+datalad get -r sourcedata/freesurfer/sub-pixar001/surf
+datalad get -r sourcedata/freesurfer/fsaverage5
+
+datalad get sub-pixar001/anat/sub-pixar001_hemi-L_pial.surf.gii
+datalad get sub-pixar001/func/sub-pixar001_task-pixar_hemi-L_space-fsaverage5_bold.func.gii
+datalad get sub-pixar001/func/sub-pixar001_task-pixar_space-fsLR_den-91k_bold.dtseries.nii
+
+# get the surface data from midnight scan club
+cd ~/brainhackschool_nibabel/data
+wget -c https://raw.githubusercontent.com/MidnightScanClub/MSCcodebase/master/Utilities/Conte69_atlas-v2.LR.32k_fs_LR.wb/Conte69.L.inflated.32k_fs_LR.surf.gii 
+
+# get the tutorial
+cd ~/brainhackschool_nibabel
+wget -c https://raw.githubusercontent.com/school-brainhack/school-brainhack.github.io/tree/main/content/en/modules/nibabel/Pipfile
+wget -c https://raw.githubusercontent.com/school-brainhack/school-brainhack.github.io/tree/main/content/en/modules/nibabel/Nibabel.py
+
+# make a temporary directory to dump things
+mkdir ~/brainhackschool_nibabel/data/tmp
+```
+
+### Set up environment
+
+You will need to create the environment with the `Pipfile`.
+
+```
+pip install pipenv  # You need pipenv to create an environmnet
+cd ~/brainhackschool_nibabel/
+pipenv install Pipfile  # install the environment
```

Comment:
Issues up to here:

1.  Since I couldn't use wget for the Pipfile I just created a file and copy & pasted the contents in, then create the environment. 
2. But, I had to download python3.8 separately before `pipenv run jupyter-notebook` would run, I don't know if that is related to just my os, or if students might have the same problem?
3. The notebook did not load properly (but that may be related to my hack at step 1):
![Screenshot 2023-05-03 at 15 31 31](https://user-images.githubusercontent.com/57987005/236025432-116e5bf9-8953-4d8d-b486-98db6a28d650.png)

Reply:
See above for problem 1

Problem two - this environment requires python3.8. Let me see if this can be loosen

Reply:
I now use `requirements.txt` to get around the problem. Tested in python3.10